enterprise/websvc.manager/nbproject/project.xml

```diff
@@ -335,6 +344,10 @@
                 <package>org.netbeans.modules.websvc.manager.spi</package>
                 <package>org.netbeans.modules.websvc.manager.util</package>
             </friend-packages>
+            <class-path-extension>
+                <binary-origin>external/jaxb-api-2.3.1.jar</binary-origin>
```

Comment:
...`jaxb.api` present as an enclosed library. It seems to me as a needless duplication.

Shouldn't the enclosed library be removed?

Reply:
Had corrected the same in latest commit